server/controller/message/communityMessage.js

```diff
@@ -0,0 +1,61 @@
+import models from '../../models';
+import { lineCheck, linkCheck } from '../../utils/messageCheck';
+
+const { User, CommunityMessage } = models;
+
+
+/**
+ * A class that handles user profile operations
+* */
+class CommunityMessageController {
+  /**
+     * post a message to the community
+     * @param {object} req - request object
+     * @param {object} res - response object
+     * @param{function} next - next function
+     * @returns {object} response object
+     *
+     */
+  static async postCommunityMessage(req, res) {
+    try {
+      const { userId } = req.user;
+      let isApproved = null;
+      const user = await User.findByPk(userId);
+      const { body } = req.body;
+      if (linkCheck(body) || lineCheck(body) === 'true') {
```

Comment:
Just having `if (linkCheck(body) || lineCheck(body))` without `lineCheck(body) === 'true'` does the trick as well